js/bybit.js

```diff
@@ -2337,29 +2840,58 @@ module.exports = class bybit extends Exchange {
         //         }
         //     }
         //
-        let trades = this.safeValue (response, 'result', {});
-        if (!Array.isArray (trades)) {
-            trades = this.safeValue2 (trades, 'dataList', 'list', []);
-        }
+        const result = this.safeValue (response, 'result', {});
+        const trades = this.safeValue (result, 'dataList', []);
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name bybit#fetchTrades
+         * @description get the list of most recent trades for a particular symbol
+         * @param {string} symbol unified symbol of the market to fetch trades for
+         * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
+         * @param {int|undefined} limit the maximum amount of trades to fetch
+         * @param {object} params extra parameters specific to the bybit api endpoint
+         * @returns {[object]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const isV3 = this.version === 'v3';
+        const isUsdcSettled = market['settle'] === 'USDC';
+        if (market['type'] === 'spot') {
+            return await this.fetchSpotTrades (symbol, since, limit, params);
```

Comment:
also, something broke fetchTrades for spot markets: 
![image](https://user-images.githubusercontent.com/43336371/201888273-0fe01319-42b7-4e49-af92-d264252139dd.png)